### PR TITLE
issue fix in listing

### PR DESCRIPTION
**Pull request by adithyadinesh0412**
Issues fixed.

review status key fix.
added review_status key for all up for review resources. 
Removed resources directly assigned to other reviewers from the list.
Removed the resources which are approved , requested for changes from the list.

### CODE REVIEW

File: src/services/resource.js [hunk 1]
```diff
@@ -882,6 +921,89 @@ module.exports = class resourceHelper {
...
+			reviewer_id: { [Op.notIn]: [loggedInUserId] },
+		}
+		const reviewsResponse = await reviewsQueries.findAll(reviewsFilter, ['resource_id'])
+		let resourceIdsToBeRemoved = []
+		if (reviewsResponse) {
+			// push resource ids to resourceIdsToBeRemoved array
+			resourceIdsToBeRemoved = reviewsResponse.map((item) => item.resource_id)
+			return resourceIdsToBeRemoved
+		}
+		return resourceIdsToBeRemoved
```
Comment (adithyadinesh0412) [inline src/services/resource.js]: remove duplicates

File: src/services/resource.js [hunk 2]
```diff
@@ -882,6 +921,89 @@ module.exports = class resourceHelper {
...
+		const reviewsFilter = {
+			resource_id: { [Op.in]: finalResourceIds },
+			status: {
+				[Op.in]: [common.REVIEW_STATUS_APPROVED, common.REVIEW_STATUS_REQUESTED_FOR_CHANGES],
+			},
+			reviewer_id: loggedInUserId,
+		}
+		const reviewsResponse = await reviewsQueries.findAll(reviewsFilter, ['resource_id'])
+		let resourceIdsToBeRemoved = []
+		if (reviewsResponse) {
```
Comment (adithyadinesh0412) [inline src/services/resource.js]: duplicate and length check

File: src/services/resource.js [hunk 3]
```diff
@@ -541,10 +541,21 @@ module.exports = class resourceHelper {
...
+					await this.findResourcesAssignedToOtherReviewers(user_id, finalResourceIds) //remove resources directly assigned to other reviewers
+				)
+
+				// fetch resources directly assigned to me
 				const assignedToMe = await this.findResourcesAssignedToReviewer(uniqueOrganizationIds, user_id)
 
 				finalResourceIds = [...finalResourceIds, ...assignedToMe]
 
+				// resources reviewer have approved , rejected or requested for change should be removed from main list
+				const resouecesCompletedMyReview = await this.findResourcesCompletedMyReview(user_id, finalResourceIds)
```
Comment (adithyadinesh0412) [inline src/services/resource.js]: change fn name

File: src/services/resource.js (hunk 2 above)
Comment (adithyadinesh0412) [reply src/services/resource.js]: No duplicates will be here as we are checking reviews table with a specific user id. One user cannot have same resources reviewing multiple times

**Review by priyanka-TL — APPROVED** [review]

**Pull request merged**